src/main/java/org/spongepowered/common/mixin/core/entity/item/MixinEntityMinecartContainer.java

```diff
@@ -48,7 +53,13 @@
 
     protected Fabric fabric = new IInventoryFabric(this);
     protected SlotCollection slots = new SlotCollection.Builder().add(this.getSizeInventory()).build();
-    protected Lens lens = new OrderedInventoryLensImpl(0, this.getSizeInventory(), 1, this.slots);
+    protected Lens lens;
+
+    @Inject(method = "<init>", at = @At("RETURN"))
```

Comment:
This will only affect the default constructor.
Is it intended ?

Reply:
So add a `*`?

Reply:
yep, `<init>*`

Reply:
Did it on github.